controllers/index.js

```diff
@@ -10,10 +10,47 @@ router.get('/', async (req, res, next) => {
     // Fetch templates
     const {templates} = await model.list(0, null);
     const filterOptions = {
-      sort: req.query.sort || 'views',
-      templateTypes: (req.query.templateTypes || 'all').split(','),
-      categories: (req.query.categories || 'all').split(',')
+      sort: (req.query.sort) ? req.query.sort.split(',') : ['views'],
+      templateTypes: (req.query.templateTypes) ? req.query.templateTypes.split(',') : ['all'],
+      categories: (req.query.categories) ? req.query.categories.split(',') : ['all']
     };
+    
+    // Sanitizing refining values
+    // Template Types
+    filterOptions.templateTypes.forEach(function(e,i){
+        if(Object.keys(enums.allowedFilterValues.templateTypes).indexOf(e)===-1) filterOptions.templateTypes.splice(i,1);
+    });
```

Comment:
Use arrow function in `forEach` - ES6 consistency.

```
.forEach((e,i) => {
  ...
});
```

Reply:
used arrows everywhere